actors/builtin/miner/monies.go

```diff
@@ -111,3 +113,25 @@ func InitialPledgeForPower(qaPower abi.StoragePower, baselinePower abi.StoragePo
 
 	return big.Add(ipBase, additionalIP)
 }
+
+// Verifies that the total unlocked balance covers the amount needed to both cover
+// the pledge requirement and repay all fee debt.  If not aborts.
```

Comment:
Maybe "Repays all fee debt and then verifies that ..... Returns an amount that must be burnt by the actor"